dev/tests/api-functional/testsuite/Magento/GraphQl/Review/CreateProductReviewsTest.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Review;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Framework\Exception\AuthenticationException;
+use Magento\Framework\Registry;
+use Magento\Integration\Api\CustomerTokenServiceInterface;
+use Magento\Review\Model\ResourceModel\Review\Collection;
+use Magento\Review\Model\ResourceModel\Review\CollectionFactory as ReviewCollectionFactory;
+use Magento\Review\Model\Review;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+/**
+ * Test coverage for adding product reviews mutation
+ */
+class CreateProductReviewsTest extends GraphQlAbstract
+{
+    /**
+     * @var CustomerTokenServiceInterface
+     */
+    private $customerTokenService;
+
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    private $customerRepository;
+
+    /**
+     * @var ReviewCollectionFactory
+     */
+    private $reviewCollectionFactory;
+
+    /**
+     * @var Registry
+     */
+    private $registry;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
```

Comment:
Please don't forget to add the return type

Reply:
Totally agree. Added.
Thank you.